droidlet/lowlevel/minecraft/pyworld/world.py

```diff
@@ -71,7 +74,21 @@ def __init__(self, opts, spec):
 
         if hasattr(opts, "world_server") and opts.world_server:
             port = getattr(opts, "port", 25565)
-            self.setup_server(port=port)
+            self.server = self.setup_server(port=port)
+
+    def start(self, tick_rate=0.01, step_rate=0.2):
```

Comment:
we don't need this anymore bc ticker.py, right?

Reply:
resolved.